app/jitsi/database.py

```diff
@@ -0,0 +1,39 @@
+from . import db
+from datetime import datetime
+from .models import User, Room, UserRoomState, UserLocation
+
+#TODO make timeout a part of config, in seconds
+USER_TIMEOUT = 30
+
+def create_user(username, avatar):
```

Comment:
I feel like these should be class methods of the `User` model, no? I think we don't really need this module and can move this all there